components/MainContent.js

```diff
@@ -0,0 +1,18 @@
+import React from "react"
+
+function MainContent(props) {
+
+//utility function to retreive data 
+    function getPageContent(selectedPage) {
+        console.log(data)
```

Comment:
We'll want to put the definition of this utility function outside of the component so that it doesn't get declared on every render.

we will want to call it on every render though so we'll put the call site inside the component.

`const data = getPageContent(props.selectedPage)`

Also, probably don't want the `console.log` in there

Reply:
Oh yeah, you'll also want to import the data from the file you made to use it here 😉